api/src/dev/server/Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN apk --no-cache add \
         libc6-compat \
         openssh-client \
         git \
+        make \
```

Comment:
Were these other added deps all required by the rainforest CLI?

Reply:
Yes